src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,166 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationException;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final RegistrationServiceImpl REGISTRATION_SERVICE
+            = new RegistrationServiceImpl();
+    private static final StorageDaoImpl STORAGE_DAO = new StorageDaoImpl();
+    private StorageDaoImpl storageDao = new StorageDaoImpl();
+
+    @BeforeEach
+    void setUp() {
+        storageDao = new StorageDaoImpl();
+        Storage.people.clear();
+    }
+
+    @Test
+    void addUser_NullLogin_ThrowsException() {
+        User user = new User();
+        user.setLogin(null);
+        user.setPassword("qwerty1234");
+        user.setAge(17);
+        assertThrows(RegistrationException.class, () -> REGISTRATION_SERVICE.register(user));
```

Comment:
The test name should follow the convention `<methodUnderTest>_<state>_<expectedBehavior>`. In this case, it could be `register_nullLogin_notOk`.